src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -66,13 +66,15 @@ public void cannotOverrideRenewalWhenLoanPolicyDoesNotExist() {
     loansFixture.checkOutByBarcode(smallAngryPlanet, jessica,
       new DateTime(2018, DateTimeConstants.APRIL, 21, 11, 21, 43));
 
+    loanPoliciesFixture.create(unknownLoanPolicyId);
     useFallbackPolicies(
       unknownLoanPolicyId,
       requestPoliciesFixture.allowAllRequestPolicy().getId(),
       noticePoliciesFixture.activeNotice().getId(),
       overdueFinePoliciesFixture.facultyStandard().getId(),
       lostItemFeePoliciesFixture.facultyStandard().getId()
     );
+    loanPoliciesFixture.cleanUp();
```

Comment:
Please can this only delete the relevant loan policy, rather than all of them. I think that better describes this scenario.

Reply:
updated